<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnection">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>21</width>
      <height>501</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>21</width>
      <height>501</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>501</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>D</string>
    </property>
    <addaction name="actionFile"/>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
